### PR TITLE
OBPIH-1351: Fixed filtering in location and person endpoints

### PR DESCRIPTION
**Pull request by awalkowiak**
https://tickets.pih-emr.org/browse/OBPIH-1351

### CODE REVIEW

File: grails-app/controllers/org/pih/warehouse/api/UserApiController.groovy [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+* which can be found in the file epl-v10.html at the root of this distribution.
+* By using this software in any fashion, you are agreeing to be bound by
+* the terms of this license.
+* You must not remove this notice, or any other, from this software.
+**/
+package org.pih.warehouse.api
+
+import grails.converters.JSON
+
+class UserApiController extends BaseDomainApiController {
```
Comment (jmiranda) [inline grails-app/controllers/org/pih/warehouse/api/UserApiController.groovy]: Change to PersonApiController

File: src/js/utils/option-utils.jsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import apiClient from './apiClient';
 
 export const debouncedUsersFetch = _.debounce((searchTerm, callback) => {
   if (searchTerm) {
-    apiClient.get(`/openboxes/api/generic/person?name=${searchTerm}`)
+    apiClient.get(`/openboxes/api/users?name=${searchTerm}`)
```
Comment (jmiranda) [inline src/js/utils/option-utils.jsx]: change to /api/persons

File: grails-app/controllers/org/pih/warehouse/api/UserApiController.groovy (hunk 1 above)
Comment (awalkowiak) [reply grails-app/controllers/org/pih/warehouse/api/UserApiController.groovy]: fixed

File: src/js/utils/option-utils.jsx (hunk 2 above)
Comment (awalkowiak) [reply src/js/utils/option-utils.jsx]: fixed

**Pull request merged**